fedimint-server/src/consensus/mod.rs

```diff
@@ -424,7 +424,9 @@ impl FedimintConsensus {
         current
     }
 
+    /// Wait for any module to send us a proposal
```

Comment:
@joschisan will remove this method soon. Might be worth dropping the commit to save him some rebase conflicts.

Reply:
Done.